src/components/LoginModal/LinkedTextInToast.tsx

```diff
@@ -0,0 +1,23 @@
+interface ILinkedTextInToast {
+  regularText: string
+  linkedText: string
+  loginPage: number
+  setPage: (p: number) => void
+}
+
+export default function LinkedTextInToast({
+  regularText,
+  linkedText,
+  loginPage,
+  setPage,
+}: ILinkedTextInToast) {
+  const text = `${regularText} `
+  return (
+    <div>
+      {text}
+      <a role="button" className="errorLink" onClick={() => setPage(loginPage)}>
```

Comment:
Prefer inline bootstrap util classes over custom stylesheets:
```suggestion
      <a role="button" className="text-decoration-none text-white" onClick={() => setPage(loginPage)}>
```